lnrpc/signrpc/signer_server.go

```diff
@@ -446,13 +446,17 @@ func (s *Server) SignMessage(ctx context.Context,
 		},
 	}
 
-	// The signature is over the sha256 hash of the message.
 	var digest [32]byte
-	copy(digest[:], chainhash.HashB(in.Msg))
+	if in.RawMsg {
+		copy(digest[:], in.Msg)
+	} else {
+		// The signature is by default over the sha256 hash of the message.
```

Comment:
nit: line too long

Reply:
It's actually within the 80 column rule. 
![image](https://user-images.githubusercontent.com/3824379/89340389-f6624180-d69f-11ea-85d5-b5ba597843a8.png)
I'm not sure if it's a good idea to split it to two rows.

Reply:
Not if you use 8 spaces for the tab character, which is the default for golang and described in our [contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#61-contribution-checklist).

Reply:
@guggero thanks! I missed this.
Fixed.